docs/installation_configuration_guide/Common.md

```diff
@@ -62,14 +64,17 @@ See [development configuration](https://github.com/TouK/nussknacker/blob/staging
 
 ## Environment variables
 
-Environment variables are described in [Installation guide](../installation/Installation.md), they are mostly helpful in the docker setup.
+Environment variables are described in the [Installation guide](../installation/Installation.md), they are mostly helpful in the docker setup.
 
 ## Conventions
 
 * We use HoCon (see [introduction](https://github.com/lightbend/config#using-hocon-the-json-superset) or [full specification](https://github.com/lightbend/config/blob/master/HOCON.md) for details) as our main configuration format. [Lightbend config library](https://github.com/lightbend/config/tree/master) is used for parsing configuration files - you can check [documentation](https://github.com/lightbend/config#standard-behavior) for details on conventions of file names and merging of configuration files.
```

Comment:
1. "HoCon" -> "HOCON"
2. "see introduction" -> "see the introduction"
3. "or full specification" -> "or the full specification"
4. can check documentation -> "can check the documentation"